shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -188,6 +188,12 @@ void FlatlandExternalViewEmbedder::SubmitFrame(
         FML_CHECK(view_mutators.total_transform ==
                   view_params.transformMatrix());
 
+        if (viewport.pending_create_viewport_callback) {
+          FML_DCHECK(view_size.fWidth || view_size.fHeight);
```

Comment:
&& instead of ||

The Flatland documentation says "/// The logical_size must have positive X and Y components."

However, flutter totally allows this to be 0,0, so we should not DCHECK here (DCHECK wont ever fire anyways since we always build in release mode).

IMO we should just print a WARNING instead and skip the callback.

Reply:
Done.